src/main/kotlin/org/rucca/cheese/model/TaskAIAdviceLearningPathsInnerDTO.kt

```
@@ -0,0 +1,27 @@
+    val description: kotlin.String,
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("followup_questions", required = true)
+    val followupQuestions: kotlin.collections.List<kotlin.String>,
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("stage")
+    val stage: kotlin.String? = null,
+    @field:Valid
+    @Schema(example = "null", description = "")
+    @get:JsonProperty("resources")
+    val resources: kotlin.collections.List<TaskAIAdviceLearningPathsInnerResourcesInnerDTO>? = null,
+) {}
```

detekt (1.23.7)

[warning] The class or object TaskAIAdviceLearningPathsInnerDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)